pallets/capacity/src/lib.rs

```diff
@@ -145,6 +146,9 @@ pub mod pallet {
 			+ sp_std::hash::Hash
 			+ MaxEncodedLen
 			+ TypeInfo;
+
+		/// How much FRQCY one unit of Capacity costs.
+		type TokenPerCapacity: Get<u32>;
```

Comment:
Maybe we can make this a Get<Perbill> fraction?

Reply:
I don't know what you mean by that.